Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM bitwalker/alpine-elixir-phoenix:1.9.4 as build
+COPY . /opt/app
+WORKDIR /opt/app
+RUN mix do local.hex --force, local.rebar --force, deps.get --only prod, deps.compile
+ENV MIX_ENV=prod
+RUN mix release orchestrate \
```

Comment:
does the same `mix do *` trick not work for the release command?

Reply:
I was having trouble with it earlier, but I think that was because of a `.dockerignore` issue. I'll take another look.

Reply:
Nope, `mix do` and `mix release` do not play well together.

Reply:
fair enough